cli/source/subcommands/format.cpp

```diff
@@ -23,19 +23,19 @@ namespace pl::cli::sub {
         static bool metaInformation = false;
         static u64 baseAddress = 0x00;
 
-        auto subcommand = app->add_subcommand("format");
+        auto subcommand = app->add_subcommand("format", "Executes the given pattern on the given file, and output the pattern data");
 
         // Add command line arguments
-        subcommand->add_option("-i,--input,INPUT_FILE", inputFilePath, "Input file")->required()->check(CLI::ExistingFile);
+        subcommand->add_option("-i,--input,INPUT_FILE", inputFilePath, "Input file to parse")->required()->check(CLI::ExistingFile);
```

Comment:
Does it really parse the binary file? Isn't there a better word for this?

Reply:
I think "parse" is fine, but we could also say "input file to extract data from"

Reply:
I like that more tbh

Reply:
Okay, i'll change it !